libraries/chain/db_block.cpp

```diff
@@ -129,77 +129,74 @@ bool database::push_block(const signed_block& new_block, uint32_t skip)
 bool database::_push_block(const signed_block& new_block)
 { try {
    uint32_t skip = get_node_properties().skip_flags;
-   if( !(skip&skip_fork_db) )
-   {
-      /// TODO: if the block is greater than the head block and before the next maitenance interval
-      // verify that the block signer is in the current set of active witnesses.
+   /// TODO: if the block is greater than the head block and before the next maitenance interval
```

Comment:
Please fix the typo while you're at it.

Reply:
Are you speaking of the three slashes? If not, I am blind.

Reply:
"maitenance"

Reply:
Yep, I just had my eyes checked too. Maybe their equipment needs calibration. Typo fixed.